### PR TITLE
Add pascales Triangle to pythonfile.py

### PR DESCRIPTION
**Pull request by awnorth**
Hey Em, 
I'm assigning you this pull request to review.
Thanks

### CODE REVIEW

File: pythonfile.py [hunk 1]
```diff
@@ -16,3 +16,11 @@ def something(number):
     return number + int(random)
 
 print(something(0))
+
+print("1")
+print("1 1")
+print("1 2 1")
```
Comment (emilyclark2009) [inline pythonfile.py:22]: # Andrew NICE JOB!
### so cool!
+ great effort
+ nice job!
+ best partner!

File: pythonfile.py [hunk 2]
```diff
@@ -16,3 +16,11 @@ def something(number):
...
+print("1 3 3 1")
+print("1 4 6 4 1")
+print("1 5 10 10 5 1")
+print("1 6 15 20 15 6 1")
```
Comment (emilyclark2009) [inline pythonfile.py:26]: # Needy

**Review by emilyclark2009 — APPROVED** [review]
Hi Andrew,
I looked over your code and it looks great. I left a couple of comments where I thought you really excelled.
-Em

**Comment by awnorth** [comment]
Merging Pascals Triangle to the main

**Pull request merged**